modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -1314,14 +1334,13 @@ private CompletableFuture<Void> createTableLocally(
                 return failedFuture(e);
             }
 
-            startedTables.put(tableId, table);
-
             return allOf(localPartsUpdateFuture, tablesByIdFuture).thenComposeAsync(ignore -> inBusyLock(busyLock, () -> {
                         if (onNodeRecovery) {
                             SchemaRegistry schemaRegistry = table.schemaView();
                             PartitionSet partitionSet = localPartsByTableId.get(tableId);
+                            HybridTimestamp lwm = lowWatermark.getLowWatermark();
```

Comment:
Please add a comment that the low watermark will start updating only after the node is restored and there will be no races.